config/core/resources/pingsource.yaml

```diff
@@ -70,8 +70,8 @@ spec:
       JSONPath: .metadata.creationTimestamp
   versions:
   - name: v1alpha1
-    served: true
-    storage: true
+    served: false
```

Comment:
Does this work like this? I thought you needed both v1alpha1 / v1alpha2 to be served since the storage migration reads v1alpha1 then upgrades to v1alpha2, so if you remove the serving, not sure it will work?

Reply:
`served: false` means you cannot do `kubeclt get pingsources.v1alpha1.sources.knative.dev ` but `kubeclt get pingsources.sources.knative.dev ` works because of the conversion webhook. Anyway I don't know the details but it works

Reply:
Hm, ok, well, wow, I'm really confused then :) I thought it would exactly use the pingsources.v1alpha1.sources.knative.dev (as in versioned read) of v1alpha1, then patch using v1alpha2. I think pingsources.sources.knative.dev will give you the v1alpha2 already converted. But, if you've already verified that the actual conversion happens and the storage version gets bumped, 👍

Reply:
The migration tools list all resources and perform an empty patch which triggers a migration on the K8s API server. 

I did it twice and it works.